engine/runtime-integration-tests/src/test/java/org/enso/interpreter/test/VectorBuilderTest.java

```diff
@@ -0,0 +1,209 @@
+package org.enso.interpreter.test;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.net.URI;
+import org.enso.interpreter.runtime.data.text.Text;
+import org.enso.interpreter.runtime.data.vector.ArrayLikeHelpers;
+import org.enso.interpreter.runtime.warning.Warning;
+import org.enso.interpreter.runtime.warning.WarningsLibrary;
+import org.enso.interpreter.runtime.warning.WithWarnings;
+import org.enso.test.utils.ContextUtils;
+import org.graalvm.polyglot.Source;
+import org.graalvm.polyglot.Value;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+public class VectorBuilderTest {
+  @ClassRule public static final ContextUtils ctx = ContextUtils.createDefault();
+  private static Value create;
+
+  @BeforeClass
+  public static void initCreate() throws Exception {
+    final URI srcUri = new URI("memory://longs.enso");
+    final Source src =
+        Source.newBuilder(
+                "enso",
+                """
+    import Standard.Base.Data.Vector.Builder
+    from Standard.Base.Data.Vector import Vector
+
+    create pw x y z =
+        b = Builder.new propagate_warnings=pw
+        b.append x
+        b.append y
+        b.append z
+        b.to_vector
+    """,
+                "choose.enso")
+            .uri(srcUri)
+            .buildLiteral();
+
+    var module = ctx.eval(src);
+    create = module.invokeMember("eval_expression", "create");
+  }
+
+  @Test
+  public void buildVectorOfThreeLongs() throws Exception {
+    var three = create.execute(false, 42, 17, -34);
+    assertTrue("It is an array like structure", three.hasArrayElements());
+    assertEquals(3, three.getArraySize());
+    var raw = ctx.unwrapValue(three);
+    var pkg = ArrayLikeHelpers.class.getPackage().getName();
+    assertEquals("It is a long vector", pkg + ".Vector$Long", raw.getClass().getName());
```

Comment:
```suggestion
  assertThat("It is a long vector", raw.getClass().getName(), endsWith(".Vector$Long"));
```

Reply:
I was considering `asserTrue(..., contains(...)...)`, but then I decided to go for full equality check. However without spelling out the `pkg` name. Hence the concatenation.